### PR TITLE
MRSI processing package

### PR DESCRIPTION
**Pull request by brendo-k**
Created MRSI processing  package and built MRSI trajectory simulation

### CODE REVIEW

**Comment by gdevenyi** [comment]
Hi @brendo-k can you rebase your branch on the latest master HEAD so that the commit history is clean?

File: MRSI Trajectory Simulation/Simulation/CreateTrajectory.m~ [hunk 1]
```diff
@@ -0,0 +1,55 @@
+function [TrajectoryStruct] = CreateTrajectory(Trajectory, par)
```
Comment (gdevenyi) [inline MRSI Trajectory Simulation/Simulation/CreateTrajectory.m~:1]: Looks  like this file is an inappropriately committed backup copy?

File: MRSI Trajectory Simulation/Simulation/Rosette.m~ [hunk 2]
```diff
@@ -0,0 +1,100 @@
+% Rosette.m
```
Comment (gdevenyi) [inline MRSI Trajectory Simulation/Simulation/Rosette.m~:1]: Here again a backup file

File: MRSI Trajectory Simulation/Simulation/cartMRSI.m~ [hunk 3]
```diff
@@ -0,0 +1,29 @@
+function [traj, scanTime] = cartMRSI(par)
```
Comment (gdevenyi) [inline MRSI Trajectory Simulation/Simulation/cartMRSI.m~:1]: another backup file

**Review by gdevenyi — CHANGES_REQUESTED** [review]

**Comment by brendo-k** [comment]
Hi @gdevenyi did I do the rebase correctly? I also deleted the backup files in the simulation file.

**Comment by jamienear** [comment]
Thanks @brendo-k!  It looks good to my untrained eye.  I've attached a network graph showing that your branch seems to be rebased from the most recent commit on the master (07628be), so it seems right.  But I will wait to hear from @gdevenyi before I merge, in case there is something I'm missing.  

-Jamie

Screen Shot 2020-11-11 at 13 59 23

**Comment by gdevenyi** [comment]
You can see in the commit history of this PR a large number of entries which are not authored by you but instead merges on master.

Please take a look at https://stackoverflow.com/questions/56708751/github-pull-requests-showing-more-and-more-old-merges

**Comment by brendo-k** [comment]
Could a reason for the large number of commits be because this branch is behind master? I rebased my code to master and the made a PR to this branch.

**Comment by gdevenyi** [comment]
> Could a reason for the large number of commits be because this branch is behind master?

That is what rebasing is supposed to fix.

**Comment by brendo-k** [comment]
I'm a little confused. Can you help me figure out where I went wrong?
1. I made a new branch on my forked repository
2. I used `git rebase upstream master` on the new branch
3. I made a pull request from my branch to the csi_mod branch (which is 48 commits behind master)

**Comment by gdevenyi** [comment]
Okay, sorry, I see the issue here. This PR is against FID-A/csi_mod, which is very very behind master. Your branch is based off master, so its pulling in all the updated commits.

Is that intentional, or should this PR be against master?

**Comment by brendo-k** [comment]
Hi Gabe, 

No worries thanks for clearing it up! Yes, this is intentional, we didn't want to push it to master since this feature is still unfinished.

**Comment by jamienear** [comment]
Sorry, I also didn't realize that this pull request was against the csi_mod branch.  Maybe I should rebase csi_mod to master before pulling in Brenden's changes?  This way it will no longer appear that I am pulling in so many updated commits.

**Comment by gdevenyi** [comment]
@jamienear that would clear things up slightly, but its up to you :)

**Comment by jamienear** [comment]
I just re-based the csi_mod branch to master.  @gdevenyi do you think I can safely merge the pull request now?

**Comment by jamienear** [comment]
Screen Shot 2020-11-12 at 13 12 05

**Comment by gdevenyi** [comment]
Looks good :+1:

**Pull request merged**